INSTALL.md

```diff
@@ -25,23 +20,36 @@ The steps for building the Chimple application on a Windows 7 or newer are liste
     1. ANDROID_SDK_ROOT: Enter the directory from Step 1.4 above
     2. NDK_ROOT: C:\ndk\android-ndk-r12b
     3. ANT_ROOT: C:\ant\apache-ant-x.xx.x\bin
-## Build and install goa
-Goa is an application which has many games for a child to play while improving literacy.
+## Installation steps for all apps
 1. Make sure an Android device is connected with USB debug permissions set.
 2. Overwrite the directory *tools* in ANDROID_SDK_ROOT with C:\tools
 3. `cd goa`
 4. `cocos compile -p android`
 5. Copy the content of _proj.android\libs_ folder into _proj.android-studio\goa\src\main\jniLibs_ folder. __Note__: Copy only _.so_ files.
 6. Start Android Studio and import _proj.android-studio_ folder into it.
-7. Build and run the application.
+7. Download the OBB file from the latest GitHub release and follow the instructions provided there.
+8. To change the app locale just select the desired language from the build variant.
+9. Set the `localeCode` in [LevelHelpScene.h](https://github.com/maqsoftware/Simple-Class/blob/master/goa/Classes/menu/LevelHelpScene.h) file as per the build variant.
+10. Build the project.
 
 ## Activate/Deactivate Kiosk Mode
 By default, the Kiosk mode is enabled. So all games are open. To disable Kiosk mode, set the KIOSK variable in 
 `goa/Classes/menu/ScrollableGameMapScene.cpp` to false
 
-``static const bool KIOSK = false;``
+```cpp
+static const bool KIOSK = false;
+```
 
 ## FAQs
 1. How can I resolve an error encountered after running the "setup.py" script while installing cocos?
 
-    > To resolve an error encountered after running the “setup.py” script while installing cocos, check to see whether you have Python 3 installed. If you do have Python 3 installed, uninstall it and try running setup.py again.
```

Comment:
check whether you have Python 3 installed

Reply:
I think she mentioned to remove to see in the previous sentence. The current construction of the second sentence is not correct.